libs/components/package.json

```diff
@@ -67,10 +67,12 @@
     "formik": "^2.4.3",
     "lodash": "^4.17.21",
     "mui-chips-input": "2.0.0",
+    "plotly.js": "^2.27.0",
     "react-dropzone": "^14.2.3",
     "react-file-icon": "^1.3.0",
     "react-markdown": "^8.0.7",
     "react-password-checklist": "^1.5.0",
+    "react-plotly.js": "^2.6.0",
```

Comment:
do we need both the dependencies?

Reply:
yes, `plotly.js` is a peer dependency of `react-plotly.js`.